website/client/components/guilds/discovery.vue

```diff
@@ -0,0 +1,101 @@
+<template lang="pug">
+.row
+  sidebar(@:search="updateSearch", @:filter="updateFilters")
+
+  .col-10
+    .row
```

Comment:
Hmm seems. to work although it is counter intuitive when using a grid. Concise though.